src/sage/interfaces/all.py

```diff
@@ -1,14 +1,8 @@
 # interfaces to other interpreters
 
-# import problems
-try:
-    # from maxima_lib import maxima_lib
-    from sage.interfaces.maxima import maxima, Maxima
-except ImportError:
-    pass
-
 from sage.misc.lazy_import import lazy_import
 
+lazy_import('sage.interfaces.maxima_lib', 'maxima')
```

Comment:
what happened to the uppercase one?

Reply:
> Also what's the purpose of interfaces/all? Is it exposed to the user?

These are only used to initialize the interactive sage session (what you get if you simply call `sage`) and the global namespace for the doctests.